WooCommerce/Classes/POS/Presentation/ParentProductCardView.swift

```diff
@@ -0,0 +1,81 @@
+import SwiftUI
+import struct Yosemite.POSParentProduct
+
+/// Displays a card for a parent product in POS.
+struct ParentProductCardView: View {
+    private let parentProduct: POSParentProduct
+
+    @ScaledMetric private var scale: CGFloat = 1.0
+    @Environment(\.dynamicTypeSize) var dynamicTypeSize
+
+    init(parentProduct: POSParentProduct) {
+        self.parentProduct = parentProduct
+    }
+
+    var body: some View {
+        HStack(spacing: Constants.cardSpacing) {
+            POSItemImageView(imageSource: parentProduct.productImageSource,
+                             imageSize: Constants.productCardSize * scale,
+                             scale: scale)
+            .frame(width: min(Constants.productCardSize * scale, Constants.maximumProductCardSize),
+                   height: Constants.productCardSize * scale)
```

Comment:
```suggestion
                   height: Constants.productCardSize * scale)
              .clipped()
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/14771/commits/093b9520275b549afbd3d0c4af7a053bead2860d.